doorstop/core/publisher.py

```diff
@@ -18,7 +19,13 @@
     'markdown.extensions.extra',
     'markdown.extensions.sane_lists',
     'mdx_outline',
-    'mdx_math'
+    'mdx_math',
+    PlantUMLMarkdownExtension(server='http://www.plantuml.com/plantuml',
+                              cachedir='/tmp',
```

Comment:
Will this work on Windows? Perhaps `tempfile.gettempdir()` will be more portable.

Reply:
Unfortunately, I don't have a windows system available to test it. I can change it anyway to see if it still works.